### DISCUSSION TITLE
블로그 홈에 갔을 때 전체 글에 이미지가 프로필 이미지로 표시됩니다.

### DISCUSSION THREAD

**Discussion by burninghering** [pronist/hELLO#195] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/37091602/218045470-3d909226-bd07-4baf-b76c-576588ec7bd4.png)
이미지를 없애거나 변경하고 싶은데 어떻게 하면 될까요?

**Comment by pronist** [comment, marked as answer]
@burninghering 안녕하세요.

스킨 편집에 가셔서 커버를 설정하시면 되겠습니다 :)
